packages/harvester/contracts/base/KeeperCompatibleHarvester.sol

```diff
@@ -12,238 +12,238 @@ abstract contract KeeperCompatibleHarvester is IHarvester, Ownable {
         uint256 vaultIndexToCheck; // hoisted up to be able to set newStartIndex
-        harvestInfo_ = new HarvestInfo[](vaults_.length);
+        harvestInfo = new HarvestInfo[](_vaults.length);
 
         // count the number of vaults to harvest.
-        for (uint256 offset; offset < vaults_.length; ++offset) {
+        for (uint256 offset; offset < _vaults.length; ++offset) {
             // _startIndex is where to start in the _vaultRegistry array, offset is position from start index (in other words, number of vaults we've checked so far),
             // then modulo to wrap around to the start of the array, until we've checked all vaults, or break early due to hitting gas limit
             // this logic is contained in _getCircularIndex()
             vaultIndexToCheck = UpkeepLibrary._getCircularIndex(
-                _startIndex,
+                startIndex,
                 offset,
-                vaults_.length
+                _vaults.length
             );
-            address vaultAddress = vaults_[vaultIndexToCheck];
+            address vaultAddress = _vaults[vaultIndexToCheck];
 
             (
-                bool willHarvest,
-                uint256 estimatedTxCost,
-                uint256 callRewardsAmount
+            bool willHarvest,
+            uint256 estimatedTxCost,
+            uint256 callRewardsAmount
             ) = _willHarvestVault(vaultAddress);
 
-            if (willHarvest && gasLeft >= _vaultHarvestFunctionGasOverhead) {
-                gasLeft -= _vaultHarvestFunctionGasOverhead;
-                numberOfVaultsToHarvest_ += 1;
-                harvestInfo_[offset] = HarvestInfo(
+            if (willHarvest && gasLeft >= vaultHarvestFunctionGasOverhead) {
+                gasLeft -= vaultHarvestFunctionGasOverhead;
+                numberOfVaultsToHarvest += 1;
+                harvestInfo[offset] = HarvestInfo(
                     true,
                     estimatedTxCost,
                     callRewardsAmount
                 );
             }
 
-            if (gasLeft < _vaultHarvestFunctionGasOverhead) {
+            if (gasLeft < vaultHarvestFunctionGasOverhead) {
                 break;
             }
         }
 
-        newStartIndex_ = UpkeepLibrary._getCircularIndex(
+        newStartIndex = UpkeepLibrary._getCircularIndex(
             vaultIndexToCheck,
             1,
-            vaults_.length
+            _vaults.length
         );
 
-        return (harvestInfo_, numberOfVaultsToHarvest_, newStartIndex_);
+        return (harvestInfo, numberOfVaultsToHarvest, newStartIndex);
     }
 
-    function _willHarvestVault(address vaultAddress_)
-        internal
-        view
-        returns (
-            bool willHarvestVault_,
-            uint256 estimatedTxCost_,
-            uint256 callRewardAmount_
-        )
+    function _willHarvestVault(address _vaultAddress)
+    internal
+    view
+    returns (
+        bool willHarvestVault,
+        uint256,
+        uint256
```

Comment:
Why drop the named returns?

Reply:
Theres already variables with the same name if we follow the same naming convention.